web/cypress/e2e/breadcrumbs.cy.ts

```diff
@@ -0,0 +1,125 @@
+/// <reference types="cypress" />
+
+describe('Tests for Breadcrumbs', () => {
+  beforeEach(() => {
+    cy.exec('npm run quay:seed');
+    cy.request('GET', `${Cypress.env('REACT_QUAY_APP_API_URL')}/csrf_token`)
+      .then((response) => response.body.csrf_token)
+      .then((token) => {
+        cy.loginByCSRF(token);
+      });
+  });
+
+  it('Organization list page', () => {
+    cy.visit('/organization');
+    cy.get('.pf-v5-c-breadcrumb__list').should('not.exist');
```

Comment:
it's recommended to not get by the patternfly classes since they can possibly change between patternfly updates and cause tests to randomly fail. Getting by a test-id is usually the way to go

Reply:
Hmm, I don't see a test-id for this element.